agent/src/iconfucius/cli/chat_rasa.py

```
@@ -0,0 +1,404 @@
+        env.setdefault("LLM_API_HEALTH_CHECK", "false")
+
+    cmd = [
+        "rasa", "run", "--enable-api",
+        "--port", str(port),
+        "--credentials", "credentials.yml",
+        "--endpoints", "endpoints.yml",
+    ]
+    if debug:
+        cmd.append("--debug")
+
+    return subprocess.Popen(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)